Modules/errnomodule.c

```diff
@@ -848,26 +847,6 @@ errno_exec(PyObject *module)
     add_errcode("ERFKILL", ERFKILL, "Operation not possible due to RF-kill");
 #endif
 
-    /* Solaris-specific errnos */
-#ifdef ECANCELED
-    add_errcode("ECANCELED", ECANCELED, "Operation canceled");
-#endif
-#ifdef ENOTSUP
-    add_errcode("ENOTSUP", ENOTSUP, "Operation not supported");
-#endif
-#ifdef EOWNERDEAD
```

Comment:
I haven't checked the other items in this list, but at least EOWNERDEAD is available on macOS.